sentinelhub/constants.py

```diff
@@ -342,28 +342,22 @@ def from_string(mime_type_str):
         :return: A mime type enum
         :rtype: MimeType
         """
+        guessed_extension = mimetypes.guess_extension(mime_type_str)
+        mime_type_str = guessed_extension.strip('.') if guessed_extension else mime_type_str
+
         if MimeType.has_value(mime_type_str):
             return MimeType(mime_type_str)
 
         try:
             return {
                 'tif': MimeType.TIFF,
                 'jpeg': MimeType.JPG,
-                'jpg': MimeType.JPG,
                 'hdf5': MimeType.HDF,
                 'h5': MimeType.HDF
             }[mime_type_str]
         except KeyError as exception:
             raise ValueError(f'Data format .{mime_type_str} is not supported') from exception
```

Comment:
`.{mime_type_str}` should be now changed into `{mime_type_str}` in the error message. Otherwise the message could be misleading.